### ISSUE TITLE
The projects page has changed, and my eyes are bleeding again T_T

### ISSUE THREAD

**Issue by AlexandreBonneau** [StylishThemes/GitHub-Dark#530]
* **Browser**: Firefox Dev edition 56.0b6 (64 bits)
* **Url**: https://github.com/autoNumeric/autoNumeric/projects/2
* **Operating System**: Debian stable (Linux)
* **Screenshot**: 
![github_dark_bug_projects](https://user-images.githubusercontent.com/7248877/29948173-19a31914-8e49-11e7-982b-572798bedc13.png)

* **HTML of the section where the issue occurs**:

````html
<div class="project-column position-relative d-flex flex-column overflow-hidden mr-3 height-full v-align-top rounded-1 js-project-column js-socket-channel" id="project-column-1081702" data-url="/autoNumeric/autoNumeric/projects/2/columns/1081702" data-id="1081702">

  <div class="border-bottom py-2 px-3 border-bottom clearfix js-details-container details-container Details">
      <button type="button" class="float-right js-details-target btn-octicon p-1 tooltipped tooltipped-w" aria-label="Add a note to this column" aria-expanded="false">
        <svg aria-hidden="true" class="octicon octicon-plus" height="16" version="1.1" viewBox="0 0 12 16" width="12"><path fill-rule="evenodd" d="M12 9H7v5H5V9H0V7h5V2h2v5h5z"></path></svg>
      </button>

      <button type="button" data-facebox="#edit-column-1081702" class="float-right btn-octicon p-1 tooltipped tooltipped-w" aria-label="Edit column name" aria-expanded="false">
        <svg aria-hidden="true" class="octicon octicon-pencil" height="16" version="1.1" viewBox="0 0 14 16" width="14"><path fill-rule="evenodd" d="M0 12v3h3l8-8-3-3-8 8zm3 2H1v-2h1v1h1v1zm10.3-9.3L12 6 9 3l1.3-1.3a.996.996 0 0 1 1.41 0l1.59 1.59c.39.39.39 1.02 0 1.41z"></path></svg>
      </button>

      <div id="edit-column-1081702" hidden="">
        <div class="js-column-form-container">

  <h2 class="facebox-header" data-facebox-id="facebox-header">Edit column</h2>
  <!-- '"` --><!-- </textarea></xmp> --><form accept-charset="UTF-8" action="/autoNumeric/autoNumeric/projects/2/columns/1081702" autocomplete="off" class="js-update-project-column" data-column-id="1081702" data-remote="true" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" value="✓" type="hidden"><input name="_method" value="put" type="hidden"><input name="authenticity_token" value="RPqU5gU8xdV7K9sU8Q/1hgF78C3w5M+BTOyiAm6VfG6gnLOLSXVcwZ+qHOoQIlmEABL76v4hLKs4q0QhTNQZjA==" type="hidden"></div>
    <input name="card_id" type="hidden">
    <input name="content_id" type="hidden">
    <input name="content_type" type="hidden">
    <input name="client_uid" class="js-client-uid-field" value="5dcc519159a0856d9e9539e8a1cfdf74" type="hidden">

    <dl class="form-group ">
      <dt>
        <label for="project_column_name_1081702">Column name</label>
      </dt>
      <dd>
        <input class="js-project-column-name-field form-control" name="project_column[name]" id="project_column_name_1081702" value="Ideas" placeholder="Enter a column name (To Do, In Progress, Done)" maxlength="140" autofocus="" type="text">
      </dd>
      <dd class="error">
        
      </dd>
    </dl>

    <p>
      <button class="btn btn-primary btn-block" data-disable-with="Updating column..." type="submit">
        Update column
      </button>
    </p>
</form>
    <!-- '"` --><!-- </textarea></xmp> --><form accept-charset="UTF-8" action="/autoNumeric/autoNumeric/projects/2/columns/1081702" class="js-delete-project-column" data-column-id="1081702" data-remote="true" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" value="✓" type="hidden"><input name="_method" value="delete" type="hidden"><input name="authenticity_token" value="tfvt+XtYJ9EAQteElu1cYcnLmjjCDNLmeSP8GwS9nuuJna01ZmsL3/H0LIaFTu+o3co6uZJUM5/eC1gebize7g==" type="hidden"></div>
      <button class="btn btn-danger btn-block" type="submit" data-disable-with="Updating column..." data-confirm="Are you sure?">
        Delete column
      </button>
      <input name="client_uid" class="js-client-uid-field" value="5dcc519159a0856d9e9539e8a1cfdf74" type="hidden">
</form></div>

      </div>
    <h4 class="m-0 f5 v-align-middle d-inline">
      <span class="js-project-column-name">Ideas</span>
      <span class="Counter ml-1 position-relative js-column-card-count" aria-label="Contains 0 items">0</span>
    </h4>
      <div class="project-note-form mt-3">
        <!-- '"` --><!-- </textarea></xmp> --><form accept-charset="UTF-8" action="/autoNumeric/autoNumeric/projects/2/columns/1081702/cards" class="js-project-note-form js-length-limited-input-container" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" value="✓" type="hidden"><input name="_method" value="put" type="hidden"><input name="authenticity_token" value="xjPLzf5yXYelj6mp/V0JaTPDoajbV5bMocuKfh3x6Ww5MllmK1Mmeh1LyHFu2yXp0qEFOpSOQfxWeV5gD3dXrQ==" type="hidden"></div>
          <input name="content_type" value="Note" type="hidden">
          <input name="client_uid" class="js-client-uid-field" value="5dcc519159a0856d9e9539e8a1cfdf74" type="hidden">
          <textarea name="note" required="" autofocus="" aria-label="Enter a note" class="form-control input-block js-quick-submit js-size-to-fit js-note-text js-length-limited-input" data-input-max-length="250" data-warning-length="99" data-warning-text="{{remaining}} remaining" placeholder="Enter a note"></textarea>
          <p class="js-length-limited-input-warning d-none mt-3 mb-0 text-center"></p>
          <button type="submit" class="btn btn-block mt-2" data-disable-invalid="" disabled="">Add note</button>
</form>      </div>
  </div>

  <p class="js-filtered-column-card-count mt-2 mb-0 f6 text-center text-gray-light d-none"></p>

  <div id="column-cards-1081702" class="position-relative overflow-auto p-2 js-project-column-cards js-card-drag-container height-full" data-id="1081702">
    

  </div>

    <!-- '"` --><!-- </textarea></xmp> --><form accept-charset="UTF-8" action="/autoNumeric/autoNumeric/projects/2/columns/1081702/cards" class="js-project-content-form js-project-update-card" data-remote="true" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" value="✓" type="hidden"><input name="_method" value="put" type="hidden"><input name="authenticity_token" value="zB4SW1y2OKY08/G/QfVf+QH0vbgEOcdGb+kkZtLgJXIzH4DwiZdDW4w3kGfSc3N54JYZKkvgEHaYW/B4wGabsw==" type="hidden"></div>
      <input name="client_uid" class="js-client-uid-field" value="5dcc519159a0856d9e9539e8a1cfdf74" type="hidden">
      <input name="card_id" type="hidden">
      <input name="content_id" type="hidden">
      <input name="content_type" type="hidden">
      <input name="previous_card_id" type="hidden">
      <input name="keyboard_move" type="hidden">
</form></div>
````

**Comment by Mottie** [comment]
Hi @AlexandreBonneau!

This *should* be fixed from #529. Please update your style manually.

**Comment by AlexandreBonneau** [comment]
Yes!
I though since I had the auto-update activated I had the last version ; that was without seeing that this project is updated every other hour ;P

![github_dark_bug_projects_fixed](https://user-images.githubusercontent.com/7248877/29950627-c5e8a84c-8e58-11e7-8375-109c795a202c.png)

My 👀 thanks you :)
